gnat2goto/driver/goto_utils.ads

```diff
@@ -92,7 +92,7 @@ package GOTO_Utils is
                                        A_Symbol_Table : in out Symbol_Table)
                                        return Symbol
      with Pre => (Kind (Symbol_Type) = I_Code_Type
-                  and then (Kind (Value) = I_Code_Block
+                  and then (Kind (Value) in I_Code_Block | I_Code_Assert
```

Comment:
Weakening preconditions always makes me a little nervous.  Why is does this solve the problem?

Reply:
The precondition was for an I_Code_Block, An assignment range check function is created using the above function when the first assignment is encountered in a program (it could be an initialisation expression rather than an assignment statement.  Thereafter the same check function is used for all assignments that need range checking.  Normally, the kind of the value passed to this subprogram when the range check function is first created is an I_Code_Block, but if the first range check that is required is for an aggregate assignment, the kind of the value is an I_Code_Assert.

Reply:
OK. Fair.